src/codeu/chat/database/Table.java

```diff
@@ -35,4 +44,107 @@ public S getSchema() {
     return schema;
   }
 
+  /**
+   * Find objects with a given query.
+   *
+   * @param query The query.
+   * @param values The values for the query.
+   *
+   * @return The list of DBObjects found.
+   */
+  public List<DBObject<S>> findQuery(String query, String... values) {
```

Comment:
Same as above, it is better to let the method throw SQLException and let the caller decide how to handle it.